agentMET4FOF/dashboard/Dashboard_agt_net.py

```diff
@@ -443,6 +449,50 @@ def plot_monitor_memory(n_interval):
                     y_title_offset = 0.1
                 else:
                     y_title_offset = -0.1
+
+                # Check if any metadata is present that can be used to generate axis
+                # labels.
+                if isinstance(memory_data[sender_agent], dict) and \
+                        'metadata' in memory_data[sender_agent].keys():
+                    # The metadata might be a list of metadata items each
+                    # corresponding to one element of the list of datapoints in
+                    # memory_data[sender_agent]["data"] or a single metadata element
+                    # shared corresponding to all datapoints.
+                    if isinstance(memory_data[sender_agent]["metadata"], Iterable):
+                        desc = memory_data[sender_agent]["metadata"][0]
```

Comment:
Probably this is not needed, as the metadata is always observed as at least one element list. Let's try this again with the present tutorials.

Reply:
@anupam-prasad I will check this and afterwards merge this branch to finally release your awesome work on this!

Reply:
Removed again in 079fbf9.